docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -1,20 +1,28 @@
-# Adding, Updating and Deleting Users
+---
+title: Adding, Updating and Deleting Users
+summary: This article describes how to add, update and delete users in the Run:ai platform.
+authors:
+    - Jason Novich
+date: 2023-Dec-28
+---
 
 ## Introduction
 
-The Run:ai UI allows the creation of Run:ai Users. Users are assigned levels of access to all aspects of the UI including submitting jobs on the cluster.
+The Run:ai UI allows the creation of Run:ai **local** users. Local users are users that are only in the Run:ai platform. Users are assigned levels of access to all aspects of the UI including submitting jobs on the cluster.
```

Comment:
Maybe the introduction sentences should explain that Run:ai allows the admin to manage users (not just creation), and explain what are the 2 types of users - local and SSO. 
This will set the tone for the rest of the article where we mention differences between SSO and local

Reply:
@jasonnovichRunAI ping regarding the comments above